### DISCUSSION TITLE
Loading zstd compressed tiffs

### DISCUSSION THREAD

**Discussion by cookmscott** [libvips/libvips#2646] [Help wanted]
```vips crop zstandard.tif[compression=zstd] cropped.jpg 0 0 100 100```
and
```vips crop zstandard.tif cropped.jpg 0 0 100 100```

throw the following:
```source input: ZSTD compression support is not configured```

It looks like `8.9.0` ([here](https://github.com/libvips/libvips/blob/8c668303f8048cfe76f2a684244075e4b4dd5d7b/ChangeLog#L260)) supports webp and zstd, but is that only for saving? Are you able to load zstd compressed tiffs?

**Comment by jcupitt** [comment]
Hi @cookmscott,

libvips supports zstd read and write, but the precompiled libvips binaries for windows can't include it since it is GPL.

You can compile libvips yourself and add in zstd:

https://github.com/libvips/build-win64-mxe

Though of course you'll need to be careful around licensing.

**Reply by jcupitt** [reply]
Actually I notice now it's licenced under BSD as well, I suppose it's OK. I'd open an issue on the build-win64-mxe repo and ask.
